packages/storage/src/indexed-db/index.ts

```diff
@@ -404,7 +402,7 @@ export class IndexedDb implements IndexedDbInterface {
         notesForVoting.push(
           new NotesForVotingResponse({
             noteRecord: note,
-            identityKey: new IdentityKey({ ik: bech32ToUint8Array(bech32IdentityKey) }),
+            identityKey: new IdentityKey({ ik: bech32ToDefault90(bech32IdentityKey) }),
```

Comment:
is it appropriate for this deserialization to truncate at 90 chars? this is the existing behavior, but i don't think anyone noticed it's doing that.